brave/app/scripts/controllers/bitgo.js

```diff
@@ -0,0 +1,199 @@
+const ObservableStore = require('obs-store')
+const bgUtil = require('brave-bitgo-client').util
+
+const supportedCoins = {
+  btc: 'Bitcoin',
+  bch: 'Bitcoin Cash',
+  btg: 'Bitcoin Gold',
+  bsv: 'Bitcoin SV',
+  zec: 'ZCash',
+  xlm: 'Stellar',
+  xrp: 'Ripple',
+  ltc: 'Litecoin',
+  dash: 'Dash',
+  algo: 'Algorand',
+  trx: 'Tron',
+  eos: 'Eos'
+}
+
+const uint8ToArrayBuf = (array) => {
+  return array.buffer.slice(array.byteOffset, array.byteLength + array.byteOffset)
+}
+
+const promiseGetSeed = (key) => {
+  return new Promise((resolve) => {
+    if (window.chrome && window.chrome.braveWallet) {
+      window.chrome.braveWallet.getBitGoSeed(key, resolve)
+    } else {
+      // polyfill
+      const seed = new Uint8Array(32)
+      window.crypto.getRandomValues(seed)
+      resolve(uint8ToArrayBuf(seed))
+    }
+  })
+}
+
+class BitGoController {
+  constructor (opts = {}) {
+    const initState = opts.initState || {}
+    this.proxyOrigin = opts.isProduction
+      ? 'https://bitgo-proxy.brave.com/' // TODO: update this
```

Comment:
TODO can be removed, this is the actual production origin

Reply:
done